core/src/main/resources/META-INF/kotlin-jupyter-libraries/libraries.json

```diff
@@ -0,0 +1,9 @@
+{
+  "descriptors": [
+    {
+      "init": [
+        "@Suppress(\"INVISIBLE_MEMBER\", \"INVISIBLE_REFERENCE\") USE { dependencies(\"org.jetbrains.kotlinx:dataframe-jupyter:${org.jetbrains.kotlinx.dataframe.BuildConfig.VERSION}\") }"
```

Comment:
Maybe make it public or semi-hidden through some hacks? This suppress warning can break in future versions of kotlin

Reply:
it's `internal`, so basically semi-hidden, right?

Reply:
In my opinion we could also add it to a companion object of DataFrame, like `DataFrame.VERSION`; doesn't even need to be hidden